mne/beamformer/_dics.py

```diff
@@ -140,14 +309,17 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.05, label=None,
     real_filter : bool
         If True, take only the real part of the cross-spectral-density matrices
         to compute real filters as in [2]_. Default is False.
+    picks : array-like of int
+        Channel indices to use for beamforming (if None all channels
+        are used except bad channels).
```

Comment:
inverse methods don't have picks usually. They take what is available in data here evoked.
By picking evoked you avoid an extra parameter. What was the reason for adding the pick param here?